### PR TITLE
Config cleanup

### PR DESCRIPTION
**Pull request by ewels**
Ok, I've had a stab at cleaning up the nextflow config files here and deleted a lot of code 😅 

This cleanup resolves the following:

* `nextflow.config` and `conf/parameters.config` contained the exact same things, so I deleted the duplicate lines from `nextflow.config` and just added an `includeConf 'conf/parameters.config'` line there instead.
* The `-profile analysis` was not necessary so I removed it
* The `-profile test` `conf/test.config` file should only overwrite what it needs to and not repeat all parameters, I removed duplicate lines here too
* I did a bunch of whitespace formatting to make the files easier to read

As I deleted so much code here, I wanted to check that I didn't mess anything up. Nextflow has a nice subcommand `nextflow config` which parses config files and prints them. I used this to check for difference (I sorted the output lines first as if used with `-flat` option the order should make no difference).

### Original code: `nextflow run` vs `nextflow run -profile analysis`

This confirms what I suspected - that using `-profile analysis` has no effect over running without a profile.

```diff
42c42,43
< params.config_profile_description = false
---
> params.config_profile_description = 'Used to run the whole workflow'
> params.config_profile_name = 'Main profile'
```

### Original code: `nextflow run` vs Refactored: `nextflow run`

To my relief, there are zero differences in the output.

### Original code: `-profile test` vs Refactored: `-profile test`

I figured that this line wasn't meant to be there, so deliberately deleted it. There are no other differences.

```diff
485d484
< params.testtest = '/Users/ewels/GitHub/nf-core/metaboigniter/environment.yml'
```

So in conclusion, I think that these changes are safe to merge 👍🏻

### CODE REVIEW

**Comment by ewels** [comment]
Ok, I made a start going through the schema and tidying some stuff up, but it could do with a bit more work via `nf-core schema build`. The stuff that immediately springs to mind:

* Move long descriptions into the help text
* Set some parameters as hidden
* Set up some validation extras, like Enumerated Values and regex validation and stuff
* If you can be bothered - set some icons. It makes reading the docs a lot easier

**Review by PayamEmami — APPROVED** [review]
All looks fine. Thanks for the changes
